subworkflows/nf-core/bam_docounts_contamination_angsd/tests/main.nf.test.snap

```diff
@@ -0,0 +1,76 @@
+{
+    "homo_sapiens - bam bai - hapmap -- stub": {
+        "content": [
+            {
+                "0": [
+                    [
+                        {
+                            "id": "test"
+                        },
+                        "test.txt:md5,d41d8cd98f00b204e9800998ecf8427e"
+                    ]
+                ],
+                "1": [
+                    "versions.yml:md5,3bf0fb169d6ae7f35569cafd1d904f6e",
+                    "versions.yml:md5,564bdd3eda2bfc9e585010991e3451f7"
+                ],
+                "txt": [
+                    [
+                        {
+                            "id": "test"
+                        },
+                        "test.txt:md5,d41d8cd98f00b204e9800998ecf8427e"
+                    ]
+                ],
+                "versions": [
+                    "versions.yml:md5,3bf0fb169d6ae7f35569cafd1d904f6e",
+                    "versions.yml:md5,564bdd3eda2bfc9e585010991e3451f7"
+                ]
+            },
+            [
+                {
+                    "BAM_DOCOUNTS_CONTAMINATION_ANGSD:ANGSD_CONTAMINATION": {
+                        "angsd": "0.940-dirty"
+                    }
+                },
+                {
+                    "BAM_DOCOUNTS_CONTAMINATION_ANGSD:ANGSD_DOCOUNTS": {
+                        "angsd": 0.939
+                    }
+                }
+            ]
+        ],
+        "meta": {
+            "nf-test": "0.9.2",
+            "nextflow": "24.10.4"
+        },
+        "timestamp": "2025-05-17T17:55:48.028325718"
+    },
+    "homo_sapiens - bam bai - hapmap": {
+        "content": [
+            [
+                "versions.yml:md5,3bf0fb169d6ae7f35569cafd1d904f6e",
+                "versions.yml:md5,564bdd3eda2bfc9e585010991e3451f7"
+            ],
+            [
+                "-----------------",
+                "hapmap:HapMapChrX.gz counts:test.icnts.gz minMaf:0.050000 startPos:5000000 stopPos:154900000 minDepth:2 maxDepth:200 skiptrans:0 nthreads:2 seed:0 skipML:0",
+                "Method2 is subject to fluctuations due to random sampling",
+                "Seed value of 0 (zero) will use time as seed",
```

Comment:
Should we set the seed?

Reply:
This is done now, I also updated the modules test.